tools/cli/admin.go

```diff
@@ -815,6 +819,59 @@ func newDBCommands() []cli.Command {
 				AdminDBScan(c)
 			},
 		},
+		{
+			Name:  "timers",
+			Usage: "get timers from multiple shards for given time range",
+			Flags: append(getDBFlags(),
+				cli.IntFlag{
+					Name:  FlagLowerShardBound,
+					Usage: "lower bound of shard to scan (inclusive)",
+					Value: 0,
+				},
+				cli.IntFlag{
+					Name:  FlagUpperShardBound,
+					Usage: "upper bound of shard to scan (exclusive)",
+					Value: 100,
+				},
+				cli.IntFlag{
+					Name:  FlagPageSize,
+					Usage: "page size used to query db executions table",
+					Value: 500,
+				},
+				cli.IntFlag{
+					Name:  FlagRPS,
+					Usage: "target rps of database queries",
+					Value: 100,
+				},
+				cli.StringFlag{
+					Name:  FlagStartDate,
+					Usage: "start date",
+					Value: time.Now().UTC().Format(time.RFC3339),
+				},
+				cli.StringFlag{
+					Name:  FlagEndDate,
+					Usage: "end date",
+					Value: time.Now().UTC().Add(24 * time.Hour).Format(time.RFC3339),
+				},
+				cli.StringFlag{
+					Name:  FlagDomainID,
+					Usage: "filter tasks by DomainID",
+				},
+				cli.BoolFlag{
+					Name:  FlagSkipErrorMode,
+					Usage: "skip errors",
+				},
+				cli.IntSliceFlag{
+					Name:  FlagTimerType,
+					Usage: "filter by timer types, default - 2 (User Timer). Can be provided multiple times",
```

Comment:
Could you add all the timerType mappings in the usage? 
Also may also consider adding a all type (-1) and make it default, so that we don't need -ttype 0 1 2 3 4 ...

Reply:
+1 for adding a all type